Documentation/spelling_wordlist.txt

```diff
@@ -858,6 +858,8 @@ skipCRDCreation
 skipteardown
 sleepAfterInit
 snat
+socketLB
+socketLB.enabled
```

Comment:
`socketLB.enabled` should not be necessary. I see no other terms with a dot in the middle in this file.

We can also maybe remove hostServices from this file and see if Sphinx complains?

Reply:
Worked!